docs/faq/accessibility.md

```diff
@@ -20,13 +20,14 @@ nav_order: 7
 |1.3.1 Info and Relationships|Partially supports|Live chatbar|
 | 1.3.5 Identify Input Purpose|Partially support|Email chat transcript form |
 |2.1 Keyboard Accessible|Partially supports|keyboard open close announcements |
+|2.4.4 Link Purpose<br> 2.4.9 Link Purpose|supports|Messages content engagement |
```

Comment:
as you can see on the rest on this table we're usually "Partially supports"
Messages content engagement? why not directing it to links engagement